spec/requests/recipe_food_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'RecipeFood', type: :request do
```

Comment:
- [ ] Still getting that error was shown by the previous reviewer when try running tests. Kindly fix it  :point_down:
![Screenshot from 2022-05-21 14-02-24](https://user-images.githubusercontent.com/67087664/169644334-1cdcd52c-34a9-4014-a8cb-177332da46ca.png)
 
![Screenshot from 2022-05-21 13-53-42](https://user-images.githubusercontent.com/67087664/169644250-190d811d-d228-4c04-bd4a-ddd417776622.png)

Reply:
Done.

![image](https://user-images.githubusercontent.com/27709832/169646779-5358b19d-5e94-4cd3-8b42-a44d27a2a27d.png)

All tests pass.

Reply:
- Well done with fix this error.